website/blog/2021-08-12-tungsten.md

```diff
@@ -0,0 +1,343 @@
+---
+author: Vadim Chelyshov
+title: Metals v0.10.5 - Tungsten
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+We're happy to announce the release of Metals v0.10.5, which contains a lot of fixes, adds Scala 3.0.1 support,
+and brings another nice feature!
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">150</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">71</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">18</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/40?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add support for `textDocument/selectionRange`
+- [Scala 3] `Inferred Type` code action and other improvements
+- Formatter - `runner.dialect` support and `.scalafmt.conf` update suggestions
+- Dependency sources - search symbol references in the workspace
+
+## Add support for `textDocument/selectionRange`
+
+Due to the great work by [@ckipp01](https://github.com/ckipp01) Metals now implements the `textDocument/selectionRange` [LSP-method](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_selectionRange).
+It allows you to easily select an expression and expand/shrink it using shortcut combinations:
+
+
+![gif1](https://user-images.githubusercontent.com/13974112/125335989-7a5f8780-e34d-11eb-911f-42f851478737.gif)
+
+Default keybindings in VSCode:
+<table>
+  <tbody style={{"font-size": "13px"}}>
+  <tr>
+    <td></td>
+    <td>Win/Linux</td>
+    <td>macOS</td>
+  </tr>
+  <tr>
+    <td>Expand</td>
+    <td>Shift+Alt+Right</td>
+    <td>⌃⇧⌘→</td>
+  </tr>
+  <tr>
+    <td>Shrink</td>
+    <td>Shift+Alt+Left</td>
+    <td>⌃⇧⌘←</td>
+  </tr>
+  </tbody>
+</table>
+
+## [Scala3] `Inferred Type` code action and other improvements
+
+We're hard at work ensuring that the same feature-set for Scala 2 is available for Scala 3.
+This release adds Inferred Type code action support.
+
+![gif2](https://i.imgur.com/GJGFOOy.gif)
+
+In addition to other small fixes, it's worth mentioning that the `Go to Definition` functionality was improved so that in situations where there are compilation issues, you're now able to better navigate to the definition site.
```

Comment:
```suggestion
In addition to other small fixes, it's worth mentioning that the `Go to Definition` functionality was improved so that in situations, where there are compilation issues, you're now able to better navigate to the definition site.
```

Reply:
I think in English it isn't valid place for putting comma but I might be wrong.

Reply:
Hmm... might that I am forcing the polish language versions here. Either way, you can ignore it.

Reply:
Yea, I wouldn't put a `,` there.